src/SoccerStatistics.Api.Database/Repositories/Interfaces/IMatchRepository.cs

```diff
@@ -1,12 +1,10 @@
 ﻿using SoccerStatistics.Api.Database.Entities;
-using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace SoccerStatistics.Api.Database.Repositories.Interfaces
 {
     public interface IMatchRepository : IRepository
     {
         Task<Match> GetByIdAsync(uint id);
-        Task<IEnumerable<Match>> GetHistoryOfMatchesByLeagueId(uint leagueId);
```

Comment:
We dont want/need History of matches as a separate task?

Reply:
Yes! But we have to talk more about it in the meeting

Reply:
Filtering and sorting ftw